views/js/controller/list/index.js

```diff
@@ -18,23 +18,75 @@
 define([
     'jquery',
     'i18n',
+    'uri',
     'util/url',
     'ui/feedback',
     'ui/dialog/confirm',
     'layout/section',
     'css!taoBackOfficeCss/list'
-], function ($, __, urlUtil, feedback, dialogConfirm, section) {
+], function ($, __, Uri, urlUtil, feedback, dialogConfirm, section) {
```

Comment:
Can't be `uri` instead of `Uri`? For follow same camel-case approach

Reply:
I follow same-as-in-export approach, it facilitates usage search.